designsafe/static/scripts/data-depot/components/projects/pipeline-project/pipeline-project.component.html

```diff
@@ -92,18 +92,14 @@ <h3 style="text-align: center; color: orange;">
             </colgroup>
             <tr class="prj-row">
                 <td>PI</td>
-                <td>
-                    <ds-user class="prj-data" username="$ctrl.project.value.pi" format="lname">{{ $ctrl.project.value.pi }}
-                    </ds-user>
+                <td class="prj-data">
+                    <ds-user-list usernames="[$ctrl.project.value.pi]"></ds-user-list>
```

Comment:
Just curious, why is this in [] when other similar calls (like the ds-user-list below this on line 102) don't have it?

Reply:
At the moment we only have one pi on a project, and their username is in string format. Co-PIs and Team Members are in arrays of string usernames. I wrap the PI in brackets here so that the `dsUserList` directive will be able to iterate over it.